README.md

```diff
@@ -44,27 +60,27 @@ Next, check **[First Time Setup](#first-time-setup)** to continue setting up you
 
 #### Connect your store with your Omise account
 
-![Omise-WooCommerce plugin - setting page](https://user-images.githubusercontent.com/2154669/62671292-7b991480-b9c0-11e9-8627-0bc85c078365.png)
+![Omise-WooCommerce plugin - setting page](https://user-images.githubusercontent.com/2154669/68251435-b3face00-0055-11ea-9932-d2129bc3f1e7.png)
 
 Once the installation is done, the next thing that you are going to do is to connect your store with your Omise account and enable payment methods so your customers can make a purchase with an online payment.
 
 1. Log in to WordPress admin page.
 
 2. From the sidebar, go to `Omise > Settings`.
 
-3. At the Omise Settings page, you are going to set your `Public key` and `Secret key` (these keys can be found at Omise Dashboard).
+3. At the Omise Settings page, you are going to set your `Public key` and `Secret key` (these keys can be found at [Omise Dashboard](https://dashboard.omise.co/test/keys)).
 
 4. Make sure that the option `Test mode` is checked and set your Omise keys at `Public key for test` and `Secret key for test` fields if you would like to test Omise service integration.
 
 5. Click **'Save Settings'**.
 
 #### Enable payment methods
 
-> Note that Alipay, Bill Payment: Tesco, Installment, and Internet Banking payment methods are only available for merchants with a Thai-registered Omise account.
+> Note that Alipay, Bill Payment: Tesco, Installment, Internet Banking, and TrueMoney Wallet payment methods are only available for merchants with a Thai-registered Omise account.
```

Comment:
I'd like to move away from "Thai-registered" to "registered in Thailand".  But this is personal preference.

Reply:
>  Note that Alipay, Bill Payment: Tesco, Installment, Internet Banking, and TrueMoney Wallet payment methods are only available for merchants  that is registered in Thailand.

@danfowler  Sounds good?

Reply:
'...merchants that are registered in Thailand'